canonical-web-and-design/jaas.ai
# Move images to the asset server

## Done

- Remove all local images.
- Update to use asset server images.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Have a poke around, all images should be there.

## Details

- Fixes: #188.
